lib/stringify.js

```diff
@@ -19,6 +19,7 @@ var toISO = Date.prototype.toISOString;
 
 var defaults = {
     delimiter: '&',
+    doNotEncodeKeys: false,
```

Comment:
Yeah, I strongly feel that "absence" and `false` must always do the same thing when possible (some of the options in qs predate me) but it does sometimes end up with awkward option names.